vanilla/todo.js

```diff
@@ -120,6 +130,7 @@ const createTaskEditInput = (taskText) => {
 };
 
 const createStrikeThroughElement = () => {
+    // why do you need strikethrough class if you're using del?
```

Comment:
just did not want to select by element type when I have to remove it.
thought that maybe selecting by a defined class name is more elegant and readable.

Reply:
i see, valid point.